files/omeka-2.4.1/bootstrap.php

```diff
@@ -46,14 +46,7 @@
 // Define the web address constants.
 
 // Set the scheme.
-if ((isset($_SERVER['HTTPS']) && ($_SERVER['HTTPS'] == 'on' || $_SERVER['HTTPS'] === true))
-    || (isset($_SERVER['HTTP_SCHEME']) && $_SERVER['HTTP_SCHEME'] == 'https')
-    || (isset($_SERVER['SERVER_PORT']) && $_SERVER['SERVER_PORT'] == 443)
-) {
-    $base_root = 'https';
-} else {
-    $base_root = 'http';
-}
+$base_root = 'https';
```

Comment:
I think we should do something like:
`
$base_root = 'http';
if ($_SERVER['X-Forwarded-Proto'] == 'https') {
    $base_root = 'https';
}
`
The ALB should be passing back the `X-Forwarded-Proto` header.

Reply:
Understood. Changes made.